Cython/Utility/Coroutine.c

```diff
@@ -714,9 +714,14 @@ PyObject *__Pyx_Coroutine_SendEx(__pyx_CoroutineObject *self, PyObject *value, i
             PyTracebackObject *tb = (PyTracebackObject *) exc_state->exc_traceback;
             PyFrameObject *f = tb->tb_frame;
 
-            Py_XINCREF(tstate->frame);
             assert(f->f_back == NULL);
+            #if PY_VERSION_HEX >= 0x030B00A1
+            // FIXME: unclear what to do if PyThreadState_GetFrame fails
+            f->f_back = PyThreadState_GetFrame(tstate);
```

Comment:
[The docs](https://docs.python.org/3/c-api/init.html#c.PyThreadState_GetFrame) say that it returns `NULL` is there is no current frame. That seems reasonable and not a failure.

Reply:
I've updated the comment then. That should probably be updated in master too